pulp_smash/utils.py

```diff
@@ -19,6 +21,10 @@ class TaskTimedOutException(Exception):
     """Indicates that polling a task timed out."""
 
 
+class NoKnownBrokerError(Exception):
```

Comment:
Sure. I've seen it done in other well-respected projects. Doing so should also make life easier for end users should a breaking change need to be made; the number of references that need to change will be minimized if all exceptions live in a single location.